internal/starlark/starlark.go

```diff
@@ -0,0 +1,150 @@
+package starlark
+
+import (
+	"fmt"
+	"github.com/starlight-go/starlight/convert"
+	"go.starlark.net/starlark"
+	"math"
+	"net/url"
+	"regexp"
+)
+
+type StarlarkFunctionHandle struct {
+	Code   string
+	State  starlark.StringDict
```

Comment:
I don't totally get why state is stored on the function handle.. what's it doing there?

Reply:
(ie why not just use a fresh environment for each invocation)